front/app/utils/cl-intl/index.tsx

```diff
@@ -1,16 +1,19 @@
 import FormattedMessage from './FormattedMessage';
 import injectIntl from './injectIntl';
-import {
-  Messages,
-  // eslint-disable-next-line no-restricted-imports
-  FormattedMessage as OriginalFormattedMessage,
-} from 'react-intl';
+import useIntl from './useIntl';
+import { MessageDescriptor } from 'react-intl';
+
+type Props = React.ComponentProps<typeof FormattedMessage>;
 
 type IMessageInfo = {
-  message: Messages['key'];
-  values?: OriginalFormattedMessage.Props['values'];
+  message: MessageDescriptor;
+  values?: Props['values'];
 };
```

Comment:
I think we can/should get rid of IMessageInfo. I did it in the previous PR. Simplifies the codebase a bit more.

Reply:
The IMessageInfo removal happened in this PR I believe: https://github.com/CitizenLabDotCo/citizenlab/pull/2786/files

Reply:
Good point! I removed it - the PR you linked was useful, even though I couldn't use it directly 👍